docs/src/content/docs/hi/guides/sidebar.mdx

```diff
@@ -0,0 +1,541 @@
+<SidebarPreview
+	config={[
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				{
+					label: 'अवयवों',
+					link: '/hi/guides/components/',
+					badge: { text: 'नया', variant: 'default' },
+				},
+			],
+		},
+		{
+			label: 'संदर्भ',
+			badge: { text: 'पदावनत', variant: 'default' },
+			items: [
+				{
+					label: 'कॉन्फ़िगरेशन संदर्भ',
+					link: '/hi/reference/configuration/',
+				},
+				{
+					label: 'फ्रंटमैटर संदर्भ',
+					link: '/hi/reference/frontmatter/',
+				},
+				{
+					label: 'ओवरराइड संदर्भ',
+					link: '/hi/reference/overrides/',
+				},
+			],
+		},
+	]}
+/>
+
+### बैज वेरिएंट
+
+`text` और `variant` गुणों वाले ऑब्जेक्ट का उपयोग करके बैज स्टाइल को अनुकूलित करें।
+
+`text` प्रदर्शित की जाने वाली कंटेंट का प्रतिनिधित्व करता है (उदाहरण के लिए "नया")।
+`variant` गुण को निम्नलिखित मानों में से किसी एक पर सेट करके `default` स्टाइल को ओवरराइड करें, जो आपकी साइट के उच्चारण रंग का उपयोग करता है: `note`, `tip`, `danger`, `caution` या `success`।
+
+```js {10}
+starlight({
+	sidebar: [
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				// पीले "प्रयोगात्मक" बैज वाला एक लिंक।
+				{
+					label: 'अवयवों',
+					link: '/hi/guides/components/',
+					badge: { text: 'प्रयोगात्मक', variant: 'caution' },
+				},
+			],
+		},
+	],
+});
+```
+
+उपरोक्त कॉन्फ़िगरेशन निम्नलिखित साइडबार उत्पन्न करता है:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				{
+					label: 'अवयवों',
+					link: '/hi/guides/components/',
+					badge: { text: 'प्रयोगात्मक', variant: 'caution' },
+				},
+			],
+		},
+	]}
+/>
+
+## कस्टम HTML विशेषताएँ
+
+लिंक तत्व में कस्टम HTML विशेषताएँ जोड़ने के लिए लिंक में `attrs` गुण भी शामिल हो सकती है।
+
+निम्नलिखित उदाहरण में, `attrs` का उपयोग `target='_blank'` विशेषता जोड़ने के लिए किया जाता है, ताकि लिंक एक नए टैब में खुले, और लिंक लेबल को इटैलिकाइज़ करने के लिए एक कस्टम `style` विशेषता लागू करने के लिए:
+
+```js {10}
+starlight({
+	sidebar: [
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				// Astro दस्तावेज़ीकरण का एक बाहरी लिंक एक नए टैब में खुलेगा।
+				{
+					label: 'Astro दस्तावेज़ीकरण',
+					link: 'https://docs.astro.build/',
+					attrs: { target: '_blank', style: 'font-style: italic' },
+				},
+			],
+		},
+	],
+});
+```
+
+उपरोक्त कॉन्फ़िगरेशन निम्नलिखित साइडबार उत्पन्न करता है:
+
+<SidebarPreview
+	config={[
+		{
+			label: 'मार्गदर्शिका',
+			items: [
+				{
+					label: 'Astro दस्तावेज़ीकरण',
+					link: 'https://docs.astro.build/',
+					attrs: {
+						target: '_blank',
+						style: 'font-style: italic',
+					},
+				},
+			],
+		},
+	]}
+/>
+
+## अंतर्राष्ट्रीयकरण
+
+[BCP-47](https://www.w3.org/International/questions/qa-choosing-language-tags) भाषा टैग निर्दिष्ट करके प्रत्येक समर्थित भाषा के लिए लिंक या समूह लेबल का अनुवाद करने के लिए लिंक और समूह प्रविष्टियों पर `अनुवाद` संपत्ति का उपयोग करें, उदाहरण के लिए `"en"`, `"ar"`, या `"zh-CN"`, कुंजी के रूप में और अनुवादित लेबल मान के रूप में। `label` गुण का उपयोग डिफ़ॉल्ट लोकेल और बिना अनुवाद वाली भाषाओं के लिए किया जाएगा।
```

Comment:
```suggestion
[BCP-47](https://www.w3.org/International/questions/qa-choosing-language-tags) भाषा टैग निर्दिष्ट करके प्रत्येक समर्थित भाषा के लिए लिंक या समूह लेबल का अनुवाद करने के लिए लिंक और समूह प्रविष्टियों पर `translations` संपत्ति का उपयोग करें, उदाहरण के लिए `"en"`, `"ar"`, या `"zh-CN"`, कुंजी के रूप में और अनुवादित लेबल मान के रूप में। `label` गुण का उपयोग डिफ़ॉल्ट लोकेल और बिना अनुवाद वाली भाषाओं के लिए किया जाएगा।
```

One final suggestion related to the last changes: this references the property name so it should not be translated.

After that, I think it's merging time 🚀